pom.xml

```diff
@@ -263,6 +267,33 @@
         <version>${shiro.version}</version>
       </dependency>
 
+      <!-- Apache Hadoop -->
+      <dependency>
```

Comment:
I have changed some dependencies to use `hadoop-client` but it didn't include `hadoop-yarn-client`. So I left `hadoop-yarn-client`, too.